packages/core/src/utils.ts

```diff
@@ -1,25 +1,25 @@
 import { ExplorerUrl, IndexerUrl } from './constants.js';
 import { getFeeHandlerAddressesOfRoutes, getFeeHandlerTypeOfRoutes } from './multicall.js';
 import type {
-  Route,
-  TransferStatus,
-  TransferStatusResponse,
-  RouteIndexerType,
-  EnvironmentMetadata,
   Domain,
-  RouteType,
-  SygmaConfig,
   Domainlike,
   Eip1193Provider,
+  EnvironmentMetadata,
   FeeHandlerType,
   IndexerRoutesResponse,
+  Route,
+  RouteIndexerType,
+  RouteType,
+  SygmaConfig,
+  TransferStatus,
+  TransferStatusResponse,
 } from './types.js';
-import { Network, Environment } from './types.js';
+import { Environment, Network } from './types.js';
 
 import { Config } from './index.js';
 
 function getIndexerTransferUrl(
-  env: Environment = process.env.SYGMA_ENV,
+  env: Environment = process.env.SYGMA_ENV as Environment,
```

Comment:
dont need to do `as Environment` if `SYGMA_ENV` is defined in an environment declaration file